### ISSUE TITLE
Handle stubborn lights

### ISSUE THREAD

**Issue by R5fan** [hugobloem/stateful_scenes#163]
first off, I absolutely LOVE this integration, its how I always imagined HA scenes should work. Thank you!

However, I do run in to some issues in my living room, where I have >10 lights in a scene and it happens one of them doesnt switch on or off (without ever being detected as unavailable).  When turning on a scene, I can usually work around it, as I can detect the stateful scene not being on when I wanted it to be on, so I can simply apply the scene a second time (although this creates problems if the scene is turned off elsewhere, and I have no simple way to check that); but when turning off a scene (and all lights), and a light "hangs" its a lot harder, because the stateful scene switch is already off.

I see two possible solutions; either the user would need to have some way to know what the _intended_ scene state is (on or off), and then we could automate some checks and reapply if needed.

Alternatively, stateful scenes could do this for us; if you turn on or off (if configured to turn off lights) a scene and after some configurable time (say 5s), not all entities have reached their desired state, simply try again, maybe even a few times if needed for unstable networks?

**Comment by cayossarian** [comment]
Hi, Try to set the rounding higher perhaps and also debounce as some lights may need to settle.  Also, Suggest you use 1.61+ for proper debounce handling, evaluation periods which follows transition delays.  My testing seems to work but a debounce sometimes needs to be applied..  If you can't get it work share your specific configuration as to debounce, rounding, etc at the device level and any logs that may help with debug logging turned on.

**Comment by R5fan** [comment]
I dont think this has anything to do with debounce or rounding. Its the lights that occassionally just dont react to an on or off command, regardless if its sent "manually" or through stateful scenes. The real problem lies with the light or zigbee mesh, but stateful scenes could be very helpful by detecting and correcting the issue.

**Comment by dollarusername2** [comment]
Different idea, because I also got some "lazy" lights that don't always listen to me: perhaps one can define a list of "ignore this light for calculation" lights. They would be part of the scene, but afterwards ignored to check if the scene was applied correctly?
